Sources/MongoSwift/BSON/Document.swift

```diff
@@ -24,10 +24,14 @@ public class DocumentStorage {
         self.pointer = bson_new()
     }
 
-    internal init(fromPointer pointer: BSONPointer) {
+    internal init(copying pointer: BSONPointer) {
```

Comment:
Consider adding documentation here. I think you tend to do it at the call site, but it's probably useful to future readers why these are distinguished

Reply:
added docs for all three `DocumentStorage` initializers.